app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/Matchers.ts

```diff
@@ -0,0 +1,170 @@
+import { Platform } from 'react-native';
+import { color, defaultTestErrorLog } from './logMessageUtils';
+import { ComparisonMode, TestCase, TrackerCallCount } from './types';
+
+type TestValue =
+  | TrackerCallCount
+  | string
+  | Array<unknown>
+  | number
+  | bigint
+  | Record<string, unknown>;
+
+export class Matchers {
+  constructor(private currentValue: TestValue, private testCase: TestCase) {}
+
+  private assertValueIsCallTracker(
+    value: any
+  ): asserts value is TrackerCallCount {
+    if (!('name' in value && 'JS' in value && 'UI' in value)) {
+      throw Error('Invalid value');
+    }
+  }
+
+  public toBe(
+    expectedValue: TestValue,
+    comparisonMode = ComparisonMode.DISTANCE
+  ) {
+    const CONDITIONS: { [Key: string]: (e: any, v: any) => Boolean } = {
+      [ComparisonMode.STRING]: (expected, value) => {
+        return value !== expected;
+      },
+      [ComparisonMode.NUMBER]: (e, v) => {
+        return isNaN(Number(v)) ? !isNaN(e) : Number(v) !== Number(e);
```

Comment:
Don't you want to check here for +0 and -0?

Reply:
It seems like we don't need 😅 
<img width="124" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/36106620/66574b34-9aee-4c59-8733-e678e472f3ed">